apps/src/StudioApp.js

```diff
@@ -1520,13 +1529,13 @@ StudioApp.prototype.handleHideSource_ = function (options) {
   this.hideSource = true;
   var workspaceDiv = document.getElementById('codeWorkspace');
   if (!options.embed || options.level.skipInstructionsPopup) {
-    container.className = 'hide-source';
+    container.className = 'hide-source'; // TODO - change to hide-instructions?
   }
   workspaceDiv.style.display = 'none';
   document.getElementById('visualizationResizeBar').style.display = 'none';
 
   // Chrome-less share page.
-  if (this.share && options.app === 'applab') {
+  if (this.share && this.centerEmbedded) {
```

Comment:
what would be a good name for an option that meant "chromeless if mobile, wireless if not"?